backend-api/src/functions/services/clientCredentialsService/clientCredentialsService.ts

```diff
@@ -101,7 +103,7 @@ export class ClientCredentialsService
   validateAsyncCredentialRequest = (
     config: IValidateAsyncCredentialRequestConfig,
   ): Result<null> => {
-    const registeredRedirectUri = config.storedCredentials.redirect_uri;
+    const registeredRedirectUri = config.storedClientCredentials.redirect_uri;
```

Comment:
registeredClientCredentials

I don't think 'config' is the right term for this input?